src/Console/ConsoleInputArgument.php

```diff
@@ -89,6 +97,15 @@ public function __construct(
             $this->_required = $required;
             $this->_choices = $choices;
             $this->_default = $default;
+            $this->_separator = $separator;
+        }
+
+        // Separator could only be one character, not space.
+        if (isset($this->_separator)) {
```

Comment:
not needed on existing param.
```suggestion
        if ($this->_separator !== null) {
```
same below for option